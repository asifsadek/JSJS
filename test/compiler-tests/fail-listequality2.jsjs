val xs = List.range(1, 10);
val ys = List.range(1, 10);

val _ =
  if List.range(1, 10) == List.range(1, 10)
  then print("same lists")
  else print("diff lists");

