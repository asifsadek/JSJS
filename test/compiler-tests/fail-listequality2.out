Error: property 'range ( 1., 10. ) === List.range ( 1., 10. )' is not defined in module 'List'
